src/gui/options.ui

```diff
@@ -58,7 +58,7 @@
         <string>Behavior</string>
        </property>
        <property name="textAlignment">
-        <set>AlignCenter</set>
+        <set>AlignHCenter|AlignVCenter|AlignCenter</set>
```

Comment:
Why add `AlignHCenter|AlignVCenter|` here (and below)?
Is it added by Qt Designer? which version are you using?

If not, please remove it, it's redundant and most likely to be removed when another person edits this file.
http://doc.qt.io/qt-4.8/qt.html#AlignmentFlag-enum